src/components/uri/Visualization.tsx

```diff
@@ -395,13 +389,15 @@ export class VisualizationWrapped extends React.Component<
 
         const visualizationClass = await fetchVisualizationClass(this.sdk, visualizationClassUri);
         const processedVisualizationObject = fillMissingTitles(mdObject.content, locale);
-        const { afm, resultSpec } = toAfmResultSpec(processedVisualizationObject);
 
+        // getTotals might ignore subTotals
```

Comment:
What are we going to do about that?

Reply:
It was just my comment, but there is a separate ticket for subtotals, to I leave this to BB

Reply:
I removed the comment